migrations/sqls/20240228103534-add-user-identity-verified-up.sql

```diff
@@ -0,0 +1,7 @@
+ALTER TABLE users ADD COLUMN identity_verified boolean DEFAULT false;
+
+CREATE TABLE IF NOT EXISTS referrings (
+  referred_identity_id uuid PRIMARY KEY NOT NULL REFERENCES identities(id),
+  referred_by_id uuid NOT NULL REFERENCES users(id),
```

Comment:
I refer this to identity too , to make it more general cause referring by org would be possible in future some this put BE open up to do that too if it's been needed